afs/config_handler.py

```diff
@@ -59,21 +59,17 @@ def set_kernel_gateway(self, REQUEST, flow_json_file=None, env_obj={}):
         if not flow_id:
             raise ValueError('Flow id can not be empty')
 
-        afs_url = self.headers.get('Afs_url')
-        if afs_url:
-            os.environ['afs_url'] = afs_url
 
-        instance_id = self.headers.get('Instance_id')
-        if instance_id:
-            os.environ['instance_id'] = instance_id
-
-        auth_code = self.headers.get('Auth_code')
-        if auth_code:
-            os.environ['auth_code'] = auth_code
-
-        workspace_id = self.headers.get('Workspace_id')
-        if workspace_id:
-            os.environ['workspace_id'] = workspace_id
+        if self.headers.get('Afs_url') and self.headers.get('Instance_id') and self.headers.get('Auth_code') and self.headers.get('Workspace_id'):
```

Comment:
Consider this implement:
```python
required_headers = ['Afs_url', 'Instance_id', 'Auth_code', 'Workspace_id']
required_headers = {key: self.headers.get(key) for key in required_headers}
if not all(required_headers.values()):
    raise ValueError()
os.environ.update(required_headers)
```

I think this will be more flexible if we need to add more variable in the future.

Reply:
@chenjr0719 provides a great solution.